uqcsbot/gaming.py

```diff
@@ -0,0 +1,241 @@
+                                parameters["subranks"][friendlyname] = f"{position:d}{suffix:s}"
+
+            # sets the discription
+            elif tag == "description":
+                parameters["description"] = tag_text
+            # sets the thumbnail
+            elif tag == "image":
+                parameters["image"] = tag_text
+            # sets the minimum playing time
+            elif tag == "minplaytime":
+                parameters["min_time"] = tag_value
+            elif tag == "maxplaytime":
+                parameters["max_time"] = tag_value
+
+        return parameters
+
+    @classmethod
+    def format_board_game_parameters(self, parameters: dict) -> discord.Embed:
+        embed = discord.Embed(title=parameters.get("name", ":question:"))
+        embed.add_field(name="Summary", inline=False,
+                        value=(f"A board game for {parameters.get('min_players', ':question:'):d}"
+                               + f" to {parameters.get('max_players', ':question:'):d} players,"
+                               + " with a playing time of "
+                               + f" {parameters.get('min_time', ':question:'):s} minutes"
+                               + ("" if parameters.get('min_time') == parameters.get('max_time')
+                                  else f" to {parameters.get('max_time', ':question:'):s} minutes")
+                               + ".\n"
+                               f"Rated {parameters.get('score', ':question:'):s}/10 by"
+                               + f" {parameters.get('users', ':question:'):s} users.\n"
+                               f"Ranked {parameters.get('rank', ':question:'):s} overall "
+                               + "on _Board Game Geek_.\n"
+                               + "".join(f"• Ranked {value:s} in the {key:s} genre.\n"
+                                         for key, value in parameters.get("subranks", {}).items()) +
+                               f"Categories: {', '.join(parameters.get('categories', set())):s}\n"
+                               f"Mechanics: {', '.join(parameters.get('mechanics', set())):s}\n"))
+        max_message_length = 1000
+        description = parameters.get('description', ':question:')
+        if len(description) > max_message_length:
+            description = description[:max_message_length] + u"\u2026"
+        embed.add_field(name="Description", inline=False, value=description)
+        embed.add_field(name="Board Game Geek Link", inline=False,
+                        value=f"https://boardgamegeek.com/boardgame/{parameters.get('identity'):s}")
+        embed.set_thumbnail(url=parameters.get('image'))
+        return embed
+
+    @commands.command()
+    @loading_status
+    # TODO: @loading_status
+    async def bgg(self, ctx: commands.Context, *board_game: str):
+        """
+        `!bgg board_game` - Gets the details of `board_game` from Board Game Geek
+        """
```

Comment:
The help command already injects how to use the command, we only require the description of what the command does.

Reply:
Fixed.